playground/mocks/data/idp/idx/error-session-expired.json

```diff
@@ -8,7 +8,7 @@
     "type": "array",
     "value": [
       {
-        "message": "The session has expired",
+        "message": "The session has expired.",
```

Comment:
I believe this was a typo. I checked okta-core and other mocks to update this (added a `.` at the end).

Example: `error-session-expired.json`, `error-pre-versioning-ff-session-expired.json`

Reply:
It's a typo.